lib/beanie/registry_api/registry.ex

```diff
@@ -27,11 +27,17 @@ defmodule Beanie.RegistryAPI.Registry do
     url = url(registry, path)
     Logger.debug("Fetching registry url #{url}")
 
-    response = registry.http_client.get!(
```

Comment:
There should be a way to do this without rescuing.  Look into `HTTPotion.http_client_get` (without the `!`).